docs/source/discord/resources/channel/methods/list_public_archived_threads.rst

```diff
@@ -7,13 +7,20 @@ Example
 -------
 
 .. code:: c
+   
+   void done_list_public_archived_threads(struct discord *client, void *data, const struct discord_thread_response_body *body)
+   {
+     if (body.threads)
+       for (int i = 0; body.threads[i]; ++i)
+         printf("Thread name: %s\n", body.threads[i]->name);
```

Comment:
Please check the [guide](https://gist.github.com/lcsmuller/a5f2b205c3871888656b86825db90187#example-3) for the new method of iterating our built-in array types.